.regal/config.yaml

```diff
@@ -0,0 +1,98 @@
+
```

Comment:
If you wouldn't mind it, after this is committed, please file an issue for us to fix at least these style issues:

1) line length
2) identically named tests

Reply:
Will do! 👍 

I'd suggest adding the [constant condition](https://docs.styra.com/regal/rules/bugs/constant-condition) rule to that list too, but that's obviously not my call.
The way this project sometimes use

```rego
some_rule[message] {
    true
}
```

would better be expressed as

```rego
some_rule[message]
```

Or using more recent syntax

```
some_rule contains message
```

Checking the AST of these constructs (`opa parse --format json policy.rego`) shows these are all identical construct.

Having that `true` in the rule body is harmless, _but_ there are other places where a constant condition most certainly is a mistake, and having that rule disabled won't catch those. I guess they might be easy enough to spot in reviews though.

Reply:
Actually found an [issue](https://github.com/StyraInc/regal/issues/399) in Regal while looking into this, so let's leave these rules as they are until we can bump the version to the next one including the [fix](https://github.com/StyraInc/regal/pull/400) 😅